src/Mod/Assembly/CommandCreateView.py

```diff
@@ -76,7 +76,11 @@ def Activated(self):
         Gui.addModule("CommandCreateView")  # NOLINT
         Gui.doCommand("panel = CommandCreateView.TaskAssemblyCreateView()")
         self.panel = Gui.doCommandEval("panel")
-        Gui.doCommandGui("Gui.Control.showDialog(panel)")
+        Gui.doCommandGui("dialog = Gui.Control.showDialog(panel)")
```

Comment:
I notice that this has a different syntax than the others files. I recall it comes from a PR that wanted to add macro support to the workbench. https://github.com/FreeCAD/FreeCAD/pull/16671

Perhaps it would be worth uniformizing the commands one way or the other?